src/ashley/editor/widgets.py

```diff
@@ -0,0 +1,32 @@
+"""This module contains django widgets related to Ashley WYSIWYG editor """
+
+import json
+
+from django.forms.widgets import HiddenInput
+from django.utils.safestring import mark_safe
+from django.utils.translation import gettext as _
+
+
+class DraftEditor(HiddenInput):
+    """Draft.js editor form field widget."""
+
+    template_name = "widgets/editor.html"
+    props = {
+        "placeholder": _("Please enter your message here..."),
+    }
+
+    def __init__(self, attrs=None, props=None):
+        """Allow optional extension of widget props in init."""
+        super().__init__(attrs)
+        if props:
+            self.props = self.props.copy()
```

Comment:
I don't understand why you need this since you are not going to have any reference to the original dictionary after the next instruction has run.

Reply:
It was a relic from a widget that inspired me, i removed it.